kuka_sunrise_fri_driver/launch/startup.launch.py

```diff
@@ -222,4 +225,11 @@ def generate_launch_description():
             + "/config/effort_controller_config.yaml",
         )
     )
+    launch_arguments.append(
```

Comment:
please add this new argument to the doc and also extend the started controllers list (+ information on the name of the topic and contents of the message)